src/main/java/com/contrastsecurity/http/Query.java

```diff
@@ -1,9 +1,10 @@
 package com.contrastsecurity.http;
 
-public class Query extends FilterForm{
+public class Query extends FilterForm {
```

Comment:
If at some point we intend to completely remove `FilterForm` we would have to duplicate the methods in that class. We could do that now and have `Query` _not_ extend `FilterForm` and rather just have the same methods/enums.

Another option is that since `FilterForm` only contains two enums we could reverse the organization a bit and use interfaces like so:
```suggestion
public interface Query  {
    **
       * Returns the query into url parameters
       *
       * @return the query into url parameters
       */
      public String toQuery();
}

public enum LibrariesExpandQuery implements Query {
    VULNS;
    
    @Override
    public String toQuery() {
      return name().toLowerCase();
    }
```
This would have the same duplication of code but enforce that method name be used. Then again im not super familiar with what these enums are :)

Reply:
I like it 👍